src/oas-parsing/schema/oas-schema.ts

```diff
@@ -30,13 +30,14 @@ class OASSchema {
     this.operations = [];
   }
 
-  public set client(client: Swagger) {
+  public set client(client: SwaggerClient) {
     this._client = client;
   }
 
-  private async getClient(): Promise<Swagger> {
+  // eslint-disable-next-line @typescript-eslint/require-await
+  public async getClient(): Promise<SwaggerClient> {
```

Comment:
Async is no longer needed.  I kept it because:
- removal requires updating about a dozen files with similar and cascading changes
- all other methods on this class are async

It can be removed if we really want to, but keeping it seems a little cleaner.

Reply:
I think removing it would allow us to remove the eslint disable for this function, right? If so I do think it would be nice to remove. Would it make sense to create a follow up ticket to do so? Agree this change is already touching quite a few files. Maybe it would be easier to make a change focused on that separately, and it would be easier to review?

Reply:
Created a follow up ticket for this work: https://jira.devops.va.gov/browse/API-27741